lib/actions/request_review.js

```diff
@@ -0,0 +1,42 @@
+const { Action } = require('./action')
+const _ = require('lodash')
+
+const createRequestReview = async (context, number, reviewers) => {
+  return context.github.pullRequests.createReviewRequest(
+    context.repo({ number, reviewers })
+  )
+}
+
+class Comment extends Action {
+  constructor () {
+    super()
+    this.supportedEvents = [
+      'pull_request.opened'
```

Comment:
I'm curious if there is a reason for only supporting `opened` ? Seems like it can be `pull_request.*` and `pull_request_review.*`.  Thoughts?

Reply:
we don't want to fire request review every time we have an pull_request event.

Reply:
can also cause https://developer.github.com/v3/#abuse-rate-limits

Reply:
I've fixed the code to check to make sure already requested reviewers are not being requested again so i've added this to all pull_request events